### ISSUE TITLE
Missing characters to support Unicode 14

### ISSUE THREAD

**Issue by frankrolf** [adobe-fonts/noto-emoji-svg#84]
This is not so much an issue, but a list of Emoji that would be needed to bring the support to Unicode 14:

```
🛝 001 u1f6dd PLAYGROUND SLIDE
🛞 002 u1f6de WHEEL
🛟 003 u1f6df RING BUOY
🟰 004 u1f7f0 HEAVY EQUALS SIGN
🥹 005 u1f979 FACE HOLDING BACK TEARS
🧌 006 u1f9cc TROLL
🩻 007 u1fa7b X-RAY
🩼 008 u1fa7c CRUTCH
🪩 009 u1faa9 MIRROR BALL
🪪 010 u1faaa IDENTIFICATION CARD
🪫 011 u1faab LOW BATTERY
🪬 012 u1faac HAMSA
🪷 013 u1fab7 LOTUS
🪸 014 u1fab8 CORAL
🪹 015 u1fab9 EMPTY NEST
🪺 016 u1faba NEST WITH EGGS
🫃 017 u1fac3 PREGNANT MAN
🫃🏻 018 u1fac3_1f3fb PREGNANT MAN + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫃🏼 019 u1fac3_1f3fc PREGNANT MAN + EMOJI MODIFIER FITZPATRICK TYPE-3
🫃🏽 020 u1fac3_1f3fd PREGNANT MAN + EMOJI MODIFIER FITZPATRICK TYPE-4
🫃🏾 021 u1fac3_1f3fe PREGNANT MAN + EMOJI MODIFIER FITZPATRICK TYPE-5
🫃🏿 022 u1fac3_1f3ff PREGNANT MAN + EMOJI MODIFIER FITZPATRICK TYPE-6
🫄 023 u1fac4 PREGNANT PERSON
🫄🏻 024 u1fac4_1f3fb PREGNANT PERSON + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫄🏼 025 u1fac4_1f3fc PREGNANT PERSON + EMOJI MODIFIER FITZPATRICK TYPE-3
🫄🏽 026 u1fac4_1f3fd PREGNANT PERSON + EMOJI MODIFIER FITZPATRICK TYPE-4
🫄🏾 027 u1fac4_1f3fe PREGNANT PERSON + EMOJI MODIFIER FITZPATRICK TYPE-5
🫄🏿 028 u1fac4_1f3ff PREGNANT PERSON + EMOJI MODIFIER FITZPATRICK TYPE-6
🫅 029 u1fac5 PERSON WITH CROWN
🫅🏻 030 u1fac5_1f3fb PERSON WITH CROWN + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫅🏼 031 u1fac5_1f3fc PERSON WITH CROWN + EMOJI MODIFIER FITZPATRICK TYPE-3
🫅🏽 032 u1fac5_1f3fd PERSON WITH CROWN + EMOJI MODIFIER FITZPATRICK TYPE-4
🫅🏾 033 u1fac5_1f3fe PERSON WITH CROWN + EMOJI MODIFIER FITZPATRICK TYPE-5
🫅🏿 034 u1fac5_1f3ff PERSON WITH CROWN + EMOJI MODIFIER FITZPATRICK TYPE-6
🫗 035 u1fad7 POURING LIQUID
🫘 036 u1fad8 BEANS
🫙 037 u1fad9 JAR
🫠 038 u1fae0 MELTING FACE
🫡 039 u1fae1 SALUTING FACE
🫢 040 u1fae2 FACE WITH OPEN EYES AND HAND OVER MOUTH
🫣 041 u1fae3 FACE WITH PEEKING EYE
🫤 042 u1fae4 FACE WITH DIAGONAL MOUTH
🫥 043 u1fae5 DOTTED LINE FACE
🫦 044 u1fae6 BITING LIP
🫧 045 u1fae7 BUBBLES
🫰 046 u1faf0 HAND WITH INDEX FINGER AND THUMB CROSSED
🫰🏻 047 u1faf0_1f3fb HAND WITH INDEX FINGER AND THUMB CROSSED + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫰🏼 048 u1faf0_1f3fc HAND WITH INDEX FINGER AND THUMB CROSSED + EMOJI MODIFIER FITZPATRICK TYPE-3
🫰🏽 049 u1faf0_1f3fd HAND WITH INDEX FINGER AND THUMB CROSSED + EMOJI MODIFIER FITZPATRICK TYPE-4
🫰🏾 050 u1faf0_1f3fe HAND WITH INDEX FINGER AND THUMB CROSSED + EMOJI MODIFIER FITZPATRICK TYPE-5
🫰🏿 051 u1faf0_1f3ff HAND WITH INDEX FINGER AND THUMB CROSSED + EMOJI MODIFIER FITZPATRICK TYPE-6
🫱 052 u1faf1 RIGHTWARDS HAND
🫱🏻 053 u1faf1_1f3fb RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫱🏻‍🫲🏼 054 u1faf1_1f3fb_200d_1faf2_1f3fc RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3
🫱🏻‍🫲🏽 055 u1faf1_1f3fb_200d_1faf2_1f3fd RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4
🫱🏻‍🫲🏾 056 u1faf1_1f3fb_200d_1faf2_1f3fe RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5
🫱🏻‍🫲🏿 057 u1faf1_1f3fb_200d_1faf2_1f3ff RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6
🫱🏼 058 u1faf1_1f3fc RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3
🫱🏼‍🫲🏻 059 u1faf1_1f3fc_200d_1faf2_1f3fb RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫱🏼‍🫲🏽 060 u1faf1_1f3fc_200d_1faf2_1f3fd RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4
🫱🏼‍🫲🏾 061 u1faf1_1f3fc_200d_1faf2_1f3fe RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5
🫱🏼‍🫲🏿 062 u1faf1_1f3fc_200d_1faf2_1f3ff RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6
🫱🏽 063 u1faf1_1f3fd RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4
🫱🏽‍🫲🏻 064 u1faf1_1f3fd_200d_1faf2_1f3fb RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫱🏽‍🫲🏼 065 u1faf1_1f3fd_200d_1faf2_1f3fc RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3
🫱🏽‍🫲🏾 066 u1faf1_1f3fd_200d_1faf2_1f3fe RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5
🫱🏽‍🫲🏿 067 u1faf1_1f3fd_200d_1faf2_1f3ff RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6
🫱🏾 068 u1faf1_1f3fe RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5
🫱🏾‍🫲🏻 069 u1faf1_1f3fe_200d_1faf2_1f3fb RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫱🏾‍🫲🏼 070 u1faf1_1f3fe_200d_1faf2_1f3fc RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3
🫱🏾‍🫲🏽 071 u1faf1_1f3fe_200d_1faf2_1f3fd RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4
🫱🏾‍🫲🏿 072 u1faf1_1f3fe_200d_1faf2_1f3ff RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6
🫱🏿 073 u1faf1_1f3ff RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6
🫱🏿‍🫲🏻 074 u1faf1_1f3ff_200d_1faf2_1f3fb RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫱🏿‍🫲🏼 075 u1faf1_1f3ff_200d_1faf2_1f3fc RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3
🫱🏿‍🫲🏽 076 u1faf1_1f3ff_200d_1faf2_1f3fd RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4
🫱🏿‍🫲🏾 077 u1faf1_1f3ff_200d_1faf2_1f3fe RIGHTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6 + ZERO WIDTH JOINER + LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5
🫲 078 u1faf2 LEFTWARDS HAND
🫲🏻 079 u1faf2_1f3fb LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫲🏼 080 u1faf2_1f3fc LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-3
🫲🏽 081 u1faf2_1f3fd LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-4
🫲🏾 082 u1faf2_1f3fe LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-5
🫲🏿 083 u1faf2_1f3ff LEFTWARDS HAND + EMOJI MODIFIER FITZPATRICK TYPE-6
🫳 084 u1faf3 PALM DOWN HAND
🫳🏻 085 u1faf3_1f3fb PALM DOWN HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫳🏼 086 u1faf3_1f3fc PALM DOWN HAND + EMOJI MODIFIER FITZPATRICK TYPE-3
🫳🏽 087 u1faf3_1f3fd PALM DOWN HAND + EMOJI MODIFIER FITZPATRICK TYPE-4
🫳🏾 088 u1faf3_1f3fe PALM DOWN HAND + EMOJI MODIFIER FITZPATRICK TYPE-5
🫳🏿 089 u1faf3_1f3ff PALM DOWN HAND + EMOJI MODIFIER FITZPATRICK TYPE-6
🫴 090 u1faf4 PALM UP HAND
🫴🏻 091 u1faf4_1f3fb PALM UP HAND + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫴🏼 092 u1faf4_1f3fc PALM UP HAND + EMOJI MODIFIER FITZPATRICK TYPE-3
🫴🏽 093 u1faf4_1f3fd PALM UP HAND + EMOJI MODIFIER FITZPATRICK TYPE-4
🫴🏾 094 u1faf4_1f3fe PALM UP HAND + EMOJI MODIFIER FITZPATRICK TYPE-5
🫴🏿 095 u1faf4_1f3ff PALM UP HAND + EMOJI MODIFIER FITZPATRICK TYPE-6
🫵 096 u1faf5 INDEX POINTING AT THE VIEWER
🫵🏻 097 u1faf5_1f3fb INDEX POINTING AT THE VIEWER + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫵🏼 098 u1faf5_1f3fc INDEX POINTING AT THE VIEWER + EMOJI MODIFIER FITZPATRICK TYPE-3
🫵🏽 099 u1faf5_1f3fd INDEX POINTING AT THE VIEWER + EMOJI MODIFIER FITZPATRICK TYPE-4
🫵🏾 100 u1faf5_1f3fe INDEX POINTING AT THE VIEWER + EMOJI MODIFIER FITZPATRICK TYPE-5
🫵🏿 101 u1faf5_1f3ff INDEX POINTING AT THE VIEWER + EMOJI MODIFIER FITZPATRICK TYPE-6
🫶 102 u1faf6 HEART HANDS
🫶🏻 103 u1faf6_1f3fb HEART HANDS + EMOJI MODIFIER FITZPATRICK TYPE-1-2
🫶🏼 104 u1faf6_1f3fc HEART HANDS + EMOJI MODIFIER FITZPATRICK TYPE-3
🫶🏽 105 u1faf6_1f3fd HEART HANDS + EMOJI MODIFIER FITZPATRICK TYPE-4
🫶🏾 106 u1faf6_1f3fe HEART HANDS + EMOJI MODIFIER FITZPATRICK TYPE-5
🫶🏿 107 u1faf6_1f3ff HEART HANDS + EMOJI MODIFIER FITZPATRICK TYPE-6
```

**Comment by hbens** [comment]
Hello, is this something you are working on currently? I would love to get Emoji 14 support in time for our [defcon presentation](https://forum.defcon.org/node/241820) next August. Else I'll try to hack support myself, or find a workaround.

**Comment by frankrolf** [comment]
Hi, this is just an informational issue, so “hack support” it is.
Note that the required SVGs do exist in Google’s repository, the work needed on this end would be adding b/w illustrations.

**Comment by aurorazeng** [comment]
This has been addressed with version 2.100

**Comment by hbens** [comment]
Incredible!

Btw, we did go for the "hack support", and could do without issues.
